lib/v2/xaut/index.js

```diff
@@ -0,0 +1,68 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    let category = ctx.params.category;
+    const dic_html = { tzgg: 'tzgg.htm', xyyw: 'xyyw.htm', mtbd: 'mtbd1.htm', xshd: 'xshd.htm' };
+    const dic_title = { tzgg: '通知公告', xyyw: '校园要闻', mtbd: '媒体播报', xshd: '学术活动' };
+
+    // 设置默认值
+    if (dic_title[category] === undefined) {
+        category = 'tzgg';
+    }
+
+    const response = await got({
+        method: 'get',
+        url: 'http://www.xaut.edu.cn/index/' + dic_html[category],
+        headers: {
+            accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8',
+            'user-agent': 'Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.106 Safari/537.36',
```

Comment:
UA should be set via env

https://github.com/DIYgod/RSSHub/blob/b5be05b1e2bdc18e6fec5ebe137cca9937175ff3/lib/config.js#L40